### DISCUSSION TITLE
Input Capture can not use the clipboard

### DISCUSSION THREAD

**Discussion by sithlord48** [flatpak/xdg-desktop-portal#1459] [General]
### Operating System

Arch

### XDG Desktop Portal version

1.18

### Desktop Environment

Plasma 6.1

### Expected Behavior

it was discussed in https://github.com/flatpak/xdg-desktop-portal/pull/852 how the input capture would use the clipboard as well. 
Like many ppl I was expecting it to work in apps like deskflow and input-leap but it does not seam to work.

### Current Behavior

Can not copy and paste between input-leap server and client when one is using wayland.

### Steps to Reproduce

1. setup input-leap on gnome 46+ or plasma 6.1
2. run a server / client for input -leap
3. try to copy and paste from server to client or client to server.

**Comment by swick** [comment]
Is support for clipboard sharing implemented in the client software, and x-d-p version you're tasting on? Why do you believe that this is a x-d-p issue?

**Reply by sithlord48** [reply]
Yes clipboard works for all our other backends

**Comment by jadahl** [comment]
It's about the input capture portal allowing hooking up to the "abstract" clipboard portal. It's currently only usable via the remote desktop portal.

**Comment by swick** [comment]
Makes sense but is a feature request so I think this is better suited as a discussion?

**Reply by jadahl** [reply]
It's now a discussion.

**Comment by marcelopio** [comment]
Related issue on input-leap

https://github.com/input-leap/input-leap/issues/1698

**Comment by tmountain** [comment]
This is affecting another application, Deskflow, a virtual KVM. I'd like to know if there's a path forward to getting this resolved. Thanks!

**Reply by swick** [reply]
I don't believe this is blocked on anything other than someone having to do the work.

**Comment by sithlord48** [comment]
This will affect any application that uses the Remote Desktop or Input Capture Portal . Currently two projects I work on are affected `Deskflow` and `Input-leap`

**Comment by sithlord48** [comment]
Just pinging here to see if there has been any movement in the direction of getting this hooked up .

**Reply by nbolton** [reply]
+1

Side note: For others reading this, there is a [$5000 bounty on Deskflow](https://github.com/deskflow/deskflow/issues/8031) for the entire work, including the work that needs to be done in this project and any work that needs to be done in compositors.

**Comment by whot** [comment]
This is now done via #1803, I think we can close this
